sdk/storage/AzureStorageBlob/Source/StorageBlobClient.swift

```diff
@@ -80,34 +80,41 @@ public final class StorageBlobClient: PipelineClient {
     ///   - options: Options used to configure the client.
     private init(
         baseUrl: URL,
-        authPolicy: Authenticating,
+        authPolicy: Authenticating?,
```

Comment:
Optional? Is this to support an anonymous client?

Reply:
Yep, exactly.

Reply:
Might it be clearer to have an Auth policy for anonymous access that does nothing? I'm thinking it would be more self-documenting than needing to read docs or intellisense to realize that passing nil results in an anonymous-access client.

Reply:
That makes sense to me. Fixed in b5e44e2